search-image-by-image/docs/user-guide/how-to-deploy-edge-orchestrator.md

```diff
@@ -5,32 +5,34 @@ Edge Orchestrator, part of Intel’s Edge Software, simplifies edge application
 * **Secure Infrastructure Management**: Offers secure and efficient remote onboarding and management of your edge node fleet across sites and geographies. Zero-trust security configuration reduces the time required to secure your edge applications.
 
 * **Deployment Orchestration and Automation**: Lets you roll out and update applications and configure infrastructure nodes across your network from a single pane of glass. Edge Orchestrator provides automated cluster orchestration and dynamic application deployment.
+
 * **Automated Deployment**: Automates the remote installation and updating of applications at scale.
 
-* **Deep Telemetry**: Gives you policy-based life cycle management and centralized visibility into your distributed edge infrastructure and deployments.
+* **Deep Telemetry**: Provides policy-based lifecycle management and centralized visibility into your distributed edge infrastructure and deployments.
+
+* **Flexible Configuration**: From organizing your physical infrastructure to managing the permutations of executing applications in a variety of runtime environments, Edge Orchestrator gives you the flexibility to define the policies, criteria, and hierarchies that make the most sense for your specific business needs.
```

Comment:
Thsi content was taken from https://docs.openedgeplatform.intel.com/edge-manage-docs/main/user_guide/index.html#user-guide so would like to keep it the same.

Reply:
Reverted